apps/amiapp_flutter/android/app/build.gradle

```diff
@@ -78,7 +68,7 @@ flutter {
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.7.21"
```

Comment:
I think we should avoid hardcoding versions, as they are error-prone and harder to track when updating,

Reply:
agreed, left it for gradle 8 changes to not complicate this one.